src/components/vault/EnhancedChatBubble.tsx

```
@@ -0,0 +1,148 @@
+    ? "rounded-lg border border-blue-200 bg-blue-50/30 p-3 dark:border-blue-800/50 dark:bg-blue-900/10"
+    : "rounded-lg border border-gray-200 bg-gray-50/30 p-3 dark:border-gray-800/50 dark:bg-gray-900/10";
+
+  // Get text style based on sender type
+  const textStyle = isAgent
+    ? "text-sm text-green-800 dark:text-green-200"
+    : isAdmin
+    ? "text-sm text-blue-800 dark:text-blue-200"
+    : "text-sm text-gray-800 dark:text-gray-200";
+    
+  // Get icon based on sender type
+  const getIcon = () => {
```

ESLint

[error] 'getIcon' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)